src/main/java/greencity/service/impl/PlaceServiceImpl.java

```diff
@@ -149,41 +118,37 @@ public Place update(PlaceUpdateDto dto) {
         placeRepo.save(updatedPlace);
 
         updateOpening(dto.getOpeningHoursList(), updatedPlace);
-        updateDiscount(dto.getDiscounts(), updatedCategory, updatedPlace);
+        updateDiscount(dto.getDiscounts(), updatedPlace);
 
         return updatedPlace;
     }
 
     /**
-     * Method for updating set of {@link Discount} and save with new {@link Category} and {@link Place}.
+     * Method for updating set of {@link DiscountValue} and save with new {@link Category} and {@link Place}.
      *
-     * @param discountDtos    - set of {@link Discount}.
-     * @param updatedCategory - {@link Category} entity.
-     * @param updatedPlace    - {@link Place} entity.
+     * @param discounts    - set of {@link DiscountValue}.
+     * @param updatedPlace - {@link Place} entity.
```

Comment:
You should write javadoc in one way - always {@inheritDoc} or always write all doc info

Reply:
be more attentive! it's a private method